### PR TITLE
[UwU] `<hr>` styling

### PR DESCRIPTION
**Pull request by fennifith**
Not specced, this is just me messing around with things

https://unicorn-utterances-git-uwu-style-hr-oceanbit.vercel.app/posts/setup-a-react-native-monorepo

### CODE REVIEW

**Comment by crutchcorn** [comment]
I want to wait for Ed's approval on this, but I **really** like the styling

**Comment by PrattiDev** [comment]
I am approving this!

**Review by crutchcorn — APPROVED** [review]

**Pull request merged**
